docs/docs/task-handling.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Task Handling
+---
+
+Marathon handles tasks in the following ways. 
```

Comment:
The logs show when Marathon reacts to a status change. It also publishes status changes on the event stream. One can also query the status of a task.